apps/mobile/src/features/send/send-sheets/send-form-btc-sheet.tsx

```diff
@@ -35,7 +41,19 @@ export function SendForm() {
         />
       }
     >
-      <Text>{t`Send form ${asset}`}</Text>
+      <SendForm
+        protocol="nativeBtc"
+        symbol="BTC"
+        availableBalance={availableBalance}
+        fiatBalance={fiatBalance}
+      >
+        <SendForm.Asset assetName={t`Bitcoin`} chain={t`Layer 1`} icon={<BtcAvatarIcon />} />
+        <SendForm.AmountField />
+        <SendForm.RecipientField />
+        <SendForm.Memo />
+        <SendForm.Numpad />
+        <SendForm.Button />
+      </SendForm>
```

Comment:
I like this approach, it gives us the flexibility to replace certain elements if some chain as a bespoke requirements.

I think we might also want to compose together, and pass in the, the schema for each instance of SendForm too. All the balance validation, decimals logic etc differ greatly.

If, as we build out more forms, we realise sendform needs tens more props to configure behaviour, I think that might be a sign we need to build out a context for each form type too.